ted_sws/mapping_suite_processor/entrypoints/cli/__init__.py

```diff
@@ -1,3 +1,5 @@
 from ted_sws.data_manager.adapters.mapping_suite_repository import TRANSFORM_PACKAGE_NAME
+from ted_sws.mapping_suite_processor.services.conceptual_mapping_reader import CONCEPTUAL_MAPPINGS_FILE_NAME
 
-CONCEPTUAL_MAPPINGS_FILE = '{mappings_path}/{mapping_suite_id}/' + TRANSFORM_PACKAGE_NAME + '/conceptual_mappings.xlsx'
+CONCEPTUAL_MAPPINGS_FILE = '{mappings_path}/{mapping_suite_id}/' + TRANSFORM_PACKAGE_NAME + '/' \
```

Comment:
I think CONCEPTUAL_MAPPINGS_FILE is ambiguous name, rename this to keep template property ( ex. CONCEPTUAL_MAPPINGS_FILE_TEMPLATE)

Reply:
no problem, captain :)